luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -399,6 +480,24 @@ public Builder withPhysicalTableFactories(Map<String, Factory<ConfigPhysicalTabl
             return this;
         }
 
+        public void setGranularityDictionary(GranularityDictionary granularityDictionary) {
```

Comment:
We should add a function for adding a single custom granularity to the already existing dictionary.
The vast majority of the time, if people are adding custom granularities, they want to add it to the
defaults, rather than replace the defaults completely.

Reply:
Added this functionality.

Reply:
Are we thinking get and modify or get and set?

Reply:
What I added is a new function `addGranularity(String name, Granularity granularity)` that get and set a new (String, Granularity) pair. 
If key collision happens it will silently update the value (Granularity).

Reply:
Is it unidiomatic to have both the setter and "adder" exposed to the user?

Reply:
The methods serve two different purposes. The setter overwrites the current dictionary, the adder adds to the current dictionary. So there's nothing wrong with
having both.